clarity/src/vm/ast/parser/v2/mod.rs

```diff
@@ -206,6 +206,16 @@ impl<'a> Parser<'a> {
         }
     }
 
+    // TODO: #4587 create default for `SymbolicExpression`, then check if mutation tests are caught for these cases:
```

Comment:
That applies here as well, and in all the other places in this PR (I'm not going to tag them all).